packages/babel/README.md

```diff
@@ -2,16 +2,25 @@
 
 [![Gitter chat](https://badges.gitter.im/webpack-blocks.svg)](https://gitter.im/webpack-blocks)
 [![JavaScript Style Guide](https://img.shields.io/badge/code%20style-standard-brightgreen.svg)](http://standardjs.com/)
-[![NPM Version](https://img.shields.io/npm/v/@webpack-blocks/babel6.svg)](https://www.npmjs.com/package/@webpack-blocks/babel6)
+[![NPM Version](https://img.shields.io/npm/v/@webpack-blocks/babel.svg)](https://www.npmjs.com/package/@webpack-blocks/babel)
 
-This is the `babel6` block providing Babel (Version 6+) configuration using the [babel-loader](https://github.com/babel/babel-loader).
+This is the `babel` block providing Babel (Version 6+) configuration using the [babel-loader](https://github.com/babel/babel-loader).
 
+## Installation
+
+```sh
+npm install --save-dev @webpack-blocks/babel
+# or
+yarn add --dev @webpack-blocks/babel
+```
+
+Additionally, make sure the `babel-core` is installed.
```

Comment:
Also added the installation paragraph to the README

Reply:
I’d prefer `npm install --save-dev babel-core @webpack-blocks/babel` instead of “ Additionally…”.

Reply:
Fixed